fastlane/Fastfile

```diff
@@ -206,6 +208,18 @@ lane :update_docs do
   end
 end
 
+def send_mac_app_ci_reminder
```

Comment:
Seems like we're using other methods here, but it would be cool to make use of `private_lane :send_mac_app_ci_reminder do` instead, so we can showcase how to make use of _fastlane_l lanes 👍